omega1s1.red

```diff
@@ -0,0 +1,31 @@
+import path
+import integer
+import univalence
+
+let S1-univ-cover (x : S1) : type =
+  S1-rec x with
+  | base => int
+  | loop i => UA int int succ-equiv i
```

Comment:
For unknown reasons it would fail if
```
UA int int succ-equiv i
```
was replaced by
```
`(V i int int succ-equiv)
```

Reply:
sounds like a bug! lol.